### DISCUSSION TITLE
Understand and exploring the Application

### DISCUSSION THREAD

**Discussion by jagpreetrahi** [nrnb/GoogleSummerOfCode#243] [General]
In this application , I understood that your application provide courses and training in biology network and also mention details of all the professor. However , in readme.md , I gather no information about application 
Can u give me brief details of this application and am i think right what I understand?

**Comment by jagpreetrahi** [comment]
I am new here ,   want to contribute my development skills with NRNB
.

**Reply by khanspers** [reply]
Hi @jagpreetrahi. We use this repo to collect project ideas for our participation in Google Summer of Code, as well as our internal training program, NRNB Academy. Feel free to browse our project ideas and complete our [NRNB Academy interest form](http://nrnb.org/academy-admissions.html) if you find something of interest, and we can go from there.

**Comment by jagpreetrahi** [comment]
Alright , but the GSOC  2024 is going on , if I'll select the idea with approach then , will I get a chance to contribute to that idea?

**Reply by khanspers** [reply]
GitHub issues corresponding to active, current GSoC 2024 projects have been closed. For the issues that are currently open, nobody is working on those.

**Reply by jagpreetrahi** [reply]
So , I want to contribute on those issue , Would you be willing to give me the opportunity to contribute to those issues? 
I  believe , I   can make significant contributions to this issue.

**Reply by jagpreetrahi** [reply]
Hey @khanspers ,I took some time to look at your application and found an issue that I'd like to share with you. And I noticed something else on your website: an intro for the NRNB team. I think it would be a great feature to add introductions for each team member, which would make that page more engaging.

Here below is a screenshot of an issue.

**Reply by jagpreetrahi** [reply]
I  am able to share SS which is PNG format so , I''ll  DM you at gmail

**Reply by khanspers** [reply]
I don't see an entry from you for the NRNB Academy admissions form. Completing this form would be the first step towards starting a project with NRNB. This would be for the projects described in the issue tracker only; we are not interested in doing any major updates to the NRNB site at this time.

**Reply by jagpreetrahi** [reply]
Hey @khanspers , I hope you are doing well , can I start contribute to your organization  , and I have seen that your website made by using only html , so  I want  add some dynamic functionalities  by using React .JS , I make sure it would look very interactive.  I will wait for your reply. 
Thank you
Have a good day!

**Comment by jagpreetrahi** [comment]
Hey Kristina , I was not able to share a SS with you at github due to their
policy , I have personally dm you for SS . And further communication carry
on github discussion
